qiita_pet/test/test_study_handlers.py

```diff
@@ -382,5 +382,36 @@ class TestEBISubmitHandler(TestHandlerBase):
     pass
 
 
+class TestDelete(TestHandlerBase):
+    def test_delete_sample_template(self):
+        response = self.post('/study/description/1',
+                             {'sample_template_id': 1,
+                              'action': 'delete_sample_template'})
+        self.assertEqual(response.code, 200)
```

Comment:
Just because the delete function works in qiita_db, doesn't mean the call works in qiita_pet. The delete function may change, breaking qiita_pet, or some other thing may change. If you test in qiita pet that the deletion actually happens, you know the functionality always works. As an example, in test_delete_raw_data, you can make sure that the QiitaDBIDError is raised when instantiating RawData(1) to make sure it was deleted by the page call.